src/main/resources/static/overview.js

```diff
@@ -210,9 +216,10 @@ function getTableRowsAsString(componentHelperArray) {
     var tableRows = '';
     for (var j = 0; j < sortedComponents.length; j++) {
         var curComp = sortedComponents[j];
-        if ((curComp.low + curComp.med + curComp.high) > 0 && curComp.comps != '') {
+        if ((curComp.low + curComp.med + curComp.high + curComp.critical) > 0 && curComp.comps != '') {
             tableRows += '<tr><td><i class="icon-qualifier-fil"></i> '
-                + curComp.name + '</td><td style="text-align:center;"><span title="High" id="highRisk">'
+                + curComp.name + '</td><td style="text-align:center;">' + '<span title="Critical" id="criticalRisk">'
```

Comment:
I'm not sure if there is CSS associated with the risk id, but you might want to double check.

Reply:
Good catch.